### ISSUE TITLE
Could not build wheels for hadolint_py

### ISSUE THREAD

**Issue by paddyroddy** [AleksaC/hadolint-py#9]
Using version `v2.12.0.3`, the `pre-commit` hook sometimes failes with
```
[INFO] Installing environment for https://github.com/AleksaC/hadolint-py.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/runner/.cache/pre-commit/repoiv9psx27/py_env-python3.12/bin/python', '-mpip', 'install', '.')
return code: 1
stdout:
    Processing /home/runner/.cache/pre-commit/repoiv9psx27
      Installing build dependencies: started
      Installing build dependencies: finished with status 'done'
      Getting requirements to build wheel: started
      Getting requirements to build wheel: finished with status 'done'
      Installing backend dependencies: started
      Installing backend dependencies: finished with status 'done'
      Preparing metadata (pyproject.toml): started
      Preparing metadata (pyproject.toml): finished with status 'done'
    Building wheels for collected packages: hadolint_py
      Building wheel for hadolint_py (pyproject.toml): started
      Building wheel for hadolint_py (pyproject.toml): finished with status 'error'
    Failed to build hadolint_py
stderr:
      error: subprocess-exited-with-error
      
      × Building wheel for hadolint_py (pyproject.toml) did not run successfully.
      │ exit code: 1
      ╰─> [5 lines of output]
          running bdist_wheel
          running build
          running build_py
          running fetch_binaries
          error: [Errno 17] File exists: 'build/temp.linux-x86_64-cpython-312'
          [end of output]
      
      note: This error originates from a subprocess, and is likely not a problem with pip.
      ERROR: Failed building wheel for hadolint_py
    ERROR: Could not build wheels for hadolint_py, which is required to install pyproject.toml-based projects
Check the log at /home/runner/.cache/pre-commit/pre-commit.log
```
This seems to happen with `v2.12.1-beta` too.

**Comment by AleksaC** [comment]
Hi @paddyroddy, thanks for reporting the issue! 

I think I found the culprit. I'll try to release the fix in the next couple of hours.

**Comment by paddyroddy** [comment]
Thanks!
